db/fixtures/miq_product_features.yml

```diff
@@ -6814,6 +6814,14 @@
         :description: Display Individual Event Streams
         :feature_type: view
         :identifier: event_streams_show
+  - :name: Delete
+    :description: Delete Custom Button
+    :feature_type: view
+    :identifier: custom_button_delete
+  - :name: Set Delete
+    :description: Delete Custom Button Set
+    :feature_type: view
+    :identifier: custom_button_set_delete
```

Comment:
These feel like they should be under something instead of at the top level.  Do we have a custom buttons node?

Reply:
I moved this part of the PR to https://github.com/ManageIQ/manageiq/pull/21346